CHANGELOG.md

```diff
@@ -9,7 +9,6 @@ CHANGELOG
 # Bugfixes
 
 # Features
-
-# Commands
+- Slarkbot now responds to you when you thank him in different ways
```

Comment:
Lets move this into features as it's not a fully fledged command like /changes or /lastmatch